lms/envs/common.py

```diff
@@ -3316,9 +3315,22 @@ def _make_locale_paths(settings):  # pylint: disable=missing-function-docstring
 
     # Notifications
     'openedx.core.djangoapps.notifications',
+
+
     'openedx_events',
+
+    # Learning Core Apps, used by v2 content libraries (content_libraries app)
+    "openedx_learning.core.components.apps.ComponentsConfig",
+    "openedx_learning.core.contents.apps.ContentsConfig",
+    "openedx_learning.core.publishing.apps.PublishingConfig",
```

Comment:
```suggestion

    # Learning Core Apps, used by v2 content libraries (content_libraries app)
    "openedx_learning.core.components",
    "openedx_learning.core.contents",
    "openedx_learning.core.publishing",
```

(non-blocking nit) Same comment